tools-v2/pkg/config/bs.go

```diff
@@ -447,6 +462,26 @@ func AddBsOpRequiredFlag(cmd *cobra.Command) {
 	AddBsStringRequiredFlag(cmd, CURVEBS_OP, fmt.Sprintf("check operator name, %s", BsAvailableValueStr(CURVEBS_OP)))
 }
 
+func AddBSChunkServerIdFlag(cmd *cobra.Command) {
+	AddBsUint32RequiredFlag(cmd, CURVEBS_CHUNK_SERVER_ID, "chunkserver id")
+}
+
+func AddBSCheckCSAliveFlag(cmd *cobra.Command) {
+	AddBsBoolOptionFlag(cmd, CURVEBS_CHECK_CSALIVE, "check chunkserver alive")
+}
+
+func AddBSCheckHealthFlag(cmd *cobra.Command) {
+	AddBsBoolOptionFlag(cmd, CURVEBS_CHECK_HEALTH, "check chunkserver health")
+}
+
+func AddBSCSOfflineFlag(cmd *cobra.Command) {
+	AddBsBoolOptionFlag(cmd, CURVEBS_CS_OFFLINE, "offline")
+}
+
+func AddBSCSUnhealthyFlag(cmd *cobra.Command) {
+	AddBsBoolOptionFlag(cmd, CURVEBS_CS_UNHEALTHY, "unhealthy")
+}
```

Comment:
Add optional to the function name, and the optional flag needs to set a default value.